### PR TITLE
Update to Camel 3.9.0 & Intellij 2021.1

### LINKED ISSUE #532: Add the support for Camel 3.0 once is released
Currently the plugin supports Camel version `2.24.2`, once Camel 3.0 is officially released, I think it will make sense to add the support for Camel 3.0.

### LINKED ISSUE #560: Support IntelliJ 2020.2
Same general issue as #545, #542, #512, #501, #492, etc.

When upgrading to the just-released IntelliJ 2020.2 release, the plugin is disabled, because it is marked as no longer compatible.

Aside from Lombok (e.g. https://github.com/mplushnikov/lombok-intellij-plugin/issues/840), this is the only plugin I see consistently having this issue with every new IntelliJ release, but perhaps I'm mostly just using 1st party plugins.

### LINKED ISSUE #576: Can't install newer plugin version (only 0.5.6 available) in IntelliJ 2020.3
I can't install Apache Camel IDEA Plugin 0.6.4 in IntelliJ 2020.3.
Installation works only for 0.5.6.

**Error:** Plugin 'Apache Camel IDEA Plugin' (version '0.6.4') is not compatible with the current version of the IDE, because it requires build 201.* or older but the current build is IU-203.5981.155

**Latest release info:**
Version: 2020.3/**Build: 203.5981.155**/1 December 2020: https://www.jetbrains.com/idea/download/

**My build info:**
IntelliJ IDEA 2020.3 (Ultimate Edition)
Build #IU-203.5981.155, built on December 1, 2020
Licensed to Dmitriy Geraschenko
Subscription is active until May 27, 2021
Runtime version: 11.0.9+11-b1145.21 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 10 10.0
GC: ParNew, ConcurrentMarkSweep
Memory: 1965M
Cores: 8
Non-Bundled Plugins: com.layoutmanager, Osmorc

### PR DESCRIPTION
**Pull request by jeremyross**
This mainly contains Camel and Intellij API changes that had to be dealt with. Unfortunately, quite a few unit tests are failing, but the corresponding plugin functionality seems to be working well. Sending this PR now because I think a lot of people can make use of it while we work on getting some of the tests updated and working. 

Resolves #576, Resolves #560, Resolves #532.

### CODE REVIEW

File: camel-idea-plugin/src/main/resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -30,6 +30,11 @@
     ]]></description>
 
   <change-notes><![CDATA[
+      v.0.6.5
```
Comment (davsclaus) [inline camel-idea-plugin/src/main/resources/META-INF/plugin.xml:33]: Lets name this v.0.7 since its a big change

**Review by davsclaus — APPROVED** [review]

**Comment by davsclaus** [comment]
Can you mark the tests that fails with ignored and create another issue in the issue tracker about to look into those failing tests. Then this PR can be merged so the build would complete.

**Comment by BoBoWalker** [comment]
I’m to fresh at git hub .I just was aware I had an account in “my name”
when I’ve never created one ,My ex gf has done so much indefinitely in my
name and such .I found the bridge ,and I knew off the rip something she was
not telling me .She  now in a “rehab” but tbh with you it’s Definitely a
sex traffic type scam that’s been created .I was too blinded by a lot man
I’m playing catch up .I appreciate any help you can lend me .

**Comment by jeremyross** [comment]
> Can you mark the tests that fails with ignored and create another issue in the issue tracker about to look into those failing tests. Then this PR can be merged so the build would complete.

@davsclaus Marked tests with @Ignore, but it looks like there may be a custom runner or something that is not taking @Ignore into account.

**Comment by Kinae** [comment]
That would be a life changer for all idea user (myself included). Thank you !

**Comment by jeremyross** [comment]
Sorry, kinda borked this PR. didn't realize it had been merged. Anyway, new commit adding back support in case people are stil on intellij 2020

**Review by davsclaus — APPROVED** [review]

**Pull request merged**